packages/autoloader/src/AutoloadGenerator.php

```diff
@@ -0,0 +1,311 @@
+<?php
+// phpcs:ignoreFile -- this is not a core file
+/**
+ * Autoloader Generator.
+ */
+namespace Jetpack\Autoloader;
+
+use Composer\Autoload\AutoloadGenerator as BaseGenerator;
+use Composer\Autoload\ClassMapGenerator;
+use Composer\Config;
+use Composer\Installer\InstallationManager;
+use Composer\IO\IOInterface;
+use Composer\Package\PackageInterface;
+use Composer\Repository\InstalledRepositoryInterface;
+use Composer\Util\Filesystem;
+
+/**
+ * Class AutoloadGenerator.
+ */
+class AutoloadGenerator extends BaseGenerator {
+
+	/**
+	 * @var IOInterface
+	 */
+	private $io;
```

Comment:
I think we don't need to redefine these attributes.

Reply:
removed.